public/site.webmanifest

```
@@ -1,19 +1,14 @@
 {
-    "name": "Start UI",
-    "short_name": "Start UI",
+    "name": "start UI",
+    "short_name": "start UI",
```

Please keep the case

```suggestion
    "name": "Start UI",
    "short_name": "Start UI",
```